pages/Magical-Gadgets.md

```diff
@@ -0,0 +1,32 @@
+The Magical Gadgets contains various gadgets with magical powers. Use your magical items to create these!
+
+## Table of contents
+----
+[Ender Backpack](https://github.com/Slimefun/Slimefun4/wiki/Ender-Backpack)  
+[Magic Eye of Ender](https://github.com/Slimefun/Slimefun4/wiki/Magic-Eye-of-Ender)  
+[Elemental Staff](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Wind](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Water](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Fire](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Storm](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Magical Zombie Pills](https://github.com/Slimefun/Slimefun4/wiki/Magical-Zombie-Pills)  
+[Infused Magnet](https://github.com/Slimefun/Slimefun4/wiki/Infused-Magnets)  
+[Soulbound Backpack](https://github.com/Slimefun/Slimefun4/wiki/Soulbound-Backpack)  
+[Reinforced Spawner](https://github.com/Slimefun/Slimefun4/wiki/Reinforced-Spawner)  
+[Scroll of Dimensional Teleposition](https://github.com/Slimefun/Slimefun4/wiki/Scroll-of-Dimensional-Teleposition)  
+[Tome of Knowledge Sharing](https://github.com/Slimefun/Slimefun4/wiki/Tome-of-Knowledge-Sharing)  
+[Flask of Knowledge](https://github.com/Slimefun/Slimefun4/wiki/Flask-of-Knowledge)  
+[Ancient Altar](https://github.com/Slimefun/Slimefun4/wiki/Ancient-altar)  
+[Infernal Bonemeal](https://github.com/Slimefun/Slimefun4/wiki/Infernal-Bonemeal)  
+[Elytra Scale](https://github.com/Slimefun/Slimefun4/wiki/Elytra-Scale)  
+[Elytra](https://github.com/Slimefun/Slimefun4/wiki/Elytra)  
+[Infused Elytra](https://github.com/Slimefun/Slimefun4/wiki/Infused-Elytra)  
+[Soulbound Elytra](https://github.com/Slimefun/Slimefun4/wiki/Soulbound-Elytra)  
+[Totem of Undying](https://github.com/Slimefun/Slimefun4/wiki/Totem-of-Undying)  
+[Rainbow Wool](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Wool)  
+[Rainbow Glass](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Glass)  
+[Rainbow Glass Pane](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Glass-Pane)  
+[Rainbow Clay](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Clay)  
+[Rainbow Concrete](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Concrete)  
+[Rainbow Glazed Terracotta](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Glazed-Terracotta)  
```

Comment:
I think one single "Rainbow-Blocks" page would be fine for this, it's all pretty much the same deal :LUL: